examples/landing_zone/main.tf

```diff
@@ -68,7 +68,7 @@ module "management_vpc" {
 module "tg_gateway_connection" {
   source                    = "terraform-ibm-modules/transit-gateway/ibm"
   version                   = "2.1.1"
-  transit_gateway_name      = "${var.prefix}-tg"
+  transit_gateway_name      = var.prefix != null ? "${var.prefix}-tg" : "tg"
```

Comment:
I would not add this support to the example. The example should always use a prefix (so we avoid name clashes during test runs in our account)